src/Features/Core/Portable/Completion/CompletionHelper.cs

```diff
@@ -211,16 +171,13 @@ public int CompareMatchResults(MatchResult matchResult1, MatchResult matchResult
             }
 
             return 0;
-        }
-
-        private static bool TagsEqual(CompletionItem item1, CompletionItem item2)
-            => TagsEqual(item1.Tags, item2.Tags);
 
-        private static bool TagsEqual(ImmutableArray<string> tags1, ImmutableArray<string> tags2)
-            => tags1 == tags2 || System.Linq.Enumerable.SequenceEqual(tags1, tags2);
+            static bool IsKeywordItem(CompletionItem item)
+                => item.Tags.Contains(WellKnownTags.Keyword);
 
-        private static bool IsKeywordItem(CompletionItem item)
-            => item.Tags.Contains(WellKnownTags.Keyword);
+            static bool TagsEqual(CompletionItem item1, CompletionItem item2)
+                => item1.Tags == item2.Tags || System.Linq.Enumerable.SequenceEqual(item1.Tags, item2.Tags);
```

Comment:
is this calling the boxiing overload of SequenceEqual?  I thought we already had a SequenceEqual for immutable arrays?

Reply:
yes.  System.Linq.ImmutableArrayExtensions.  it will also do the initial `==` check, so that can be removed.